### PR TITLE
Feature/mem 13 q xml

### PR DESCRIPTION
**Pull request by barricade86**

### CODE REVIEW

File: mem-13-Q.xml [hunk 1]
```diff
@@ -5791,17 +5893,17 @@ The slang meaning of {chong:v:2,slang}, when applied to a person, is equivalent
...
       <column name="definition_fa">Kruge [AUTOTRANSLATED]</column>
       <column name="definition_sv">Kruge</column>
-      <column name="definition_ru"></column>
+      <column name="definition_ru">Крюге</column>
       <column name="definition_zh_HK">「克魯格」</column>
       <column name="synonyms"></column>
       <column name="antonyms"></column>
       <column name="see_also"></column>
-      <column name="notes"></column>
+      <column name="notes">The transliteration comes from the subtitles of {Star Trek III:src}.[2]</column>
```
Comment (De7vID) [inline mem-13-Q.xml:5901]: This change is not correct. This note belongs under "notes_zh_HK" and does not need to be translated into Russian, as it is a reference to the Chinese transliteration.

Comment (De7vID) [reply mem-13-Q.xml:5901]: That is, unless your transliteration also comes from the subtitles of Star Trek III, in which case, the same note would be applicable.

File: mem-13-Q.xml [hunk 2]
```diff
@@ -5861,16 +5963,14 @@ The slang meaning of {chong:v:2,slang}, when applied to a person, is equivalent
...
       <column name="definition_sv">spela</column>
-      <column name="definition_ru"></column>
+      <column name="definition_ru">сыграть в игру</column>
       <column name="definition_zh_HK">玩個遊戲 [AUTOTRANSLATED]</column>
       <column name="synonyms"></column>
       <column name="antonyms"></column>
       <column name="see_also">{reH:v}, {ghet:v}, {Quj:n}</column>
       <column name="notes">The difference between this and {reH:v} is that this refers to structured play, whereas {reH:v:nolink} refers to unstructured play.[2]</column>
       <column name="notes_de"></column>
-      <column name="notes_fa"></column>
```
Comment (De7vID) [inline mem-13-Q.xml:5871]: These shouldn't have been moved.

File: mem-13-Q.xml [hunk 3]
```diff
@@ -1031,7 +1039,7 @@ Watch a lesson on this word: {YouTube video:url:http://youtu.be/auqS6FR_RDE}</co
       <column name="definition_de">gut sein</column>
       <column name="definition_fa">خوب</column>
       <column name="definition_sv">vara bra</column>
-      <column name="definition_ru"></column>
+      <column name="definition_ru">быть хорошим, быть благим, быть доброкачественным, быть полезным, быть годным/column>
```
Comment (De7vID) [inline mem-13-Q.xml:1042]: Broken tag.

Comment (barricade86) [reply mem-13-Q.xml:1042]: fixed

File: mem-13-Q.xml (hunk 2 above)
Comment (barricade86) [reply mem-13-Q.xml:5871]: fixed

File: mem-13-Q.xml (hunk 1 above)
Comment (barricade86) [reply mem-13-Q.xml:5901]: fixed

File: mem-13-Q.xml (hunk 2 above)
Comment (De7vID) [reply mem-13-Q.xml:5871]: Just so you know, no changes made after the PR has already been merged will be pulled into master. So there's no need for you to fix these (as I've already fixed them when I did the merge).

**Pull request merged**